src/main/java/uk/ac/ebi/eva/pipeline/io/VepProcess.java

```diff
@@ -110,7 +110,8 @@ public void open() throws ItemStreamException {
                 "-o", "STDOUT",
                 "--force_overwrite",
                 "--offline",
-                "--everything"
+                "--everything",
+                "--format", "ensembl"
```

Comment:
the new vep can not infer the parameter (deduce the input format based on the content), but the old one accepts it